src/index.ts

```diff
@@ -151,6 +151,8 @@ export default class HorizontalScroll extends EventEmitter {
     }
 
     private wheel = (e: WheelEvent) => {
+        if (e.ctrlKey) // Ignore scroll event if ctrl key pressed
```

Comment:
I'd suggest to make it more generic by passing a function, in which it is determined if the scroll should be ignored, to the constructor. This way we make sure to make the ignoring of the scroll event configurable.
Example: 
``` 
useHorizontal({ ignoreScroll: (e) => {
    return e.ctrlKey;
}});
```

Reply:
I've made an attempt at creating a Callback interface, and using that in the options structure.